client/src/layouts/Header.vue

```diff
@@ -250,4 +265,23 @@ export default {
 #search-result {
   position: absolute;
 }
+
+.backToProfile {
+  margin-left: 20px;
+  text-align: center;
+}
+
+.backToProfile .myProfileRouterlink {
```

Comment:
.backToProfile .myProfileRouterlink => "& .myProfileRouterlink" when inside .backToProfile block (easier for refactoring further on)